tensorbay/cli/auth.py

```diff
@@ -63,19 +63,30 @@ def _implement_auth(  # pylint: disable=too-many-arguments
 
 def _get_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool) -> None:
     if is_all:
-        for key, value in config_parser["profiles"].items():
+        try:
+            profiles = config_parser["profiles"]
+        except KeyError:
+            return
+        for key, value in profiles.items():
             _echo_formatted_profile(key, value)
         return
 
     profile_name = obj["profile_name"]
-    _echo_formatted_profile(profile_name, config_parser["profiles"][profile_name])
+    try:
+        profile = config_parser["profiles"][profile_name]
+    except KeyError:
+        error(f"Profile '{profile_name}' does not exist.")
+    _echo_formatted_profile(profile_name, profile)
 
 
 def _unset_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool) -> None:
     if is_all:
         config_parser.remove_section("profiles")
     else:
-        config_parser.remove_option("profiles", obj["profile_name"])
+        try:
+            config_parser.remove_option("profiles", obj["profile_name"])
```

Comment:
If the `config_parser` does not have the profile, `remove_option` will return `False` instead of raising `NoSectionError`.